frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx

```diff
@@ -35,7 +35,8 @@ export const CreateOCSServiceForm: React.FC<CreateOCSServiceFormProps> = ({
           isInline
         />
         <p className="co-legend co-required ceph-ocs-desc__legend">
-          Select at least 3 nodes in different failure domains you wish to use.
+          Select at least 3 nodes in different failure domains with a minimum requirements of 16
```

Comment:
```suggestion
          Select at least 3 nodes in different failure domains with a minimum requirement of 16
```

Reply:
It should be `requirements` only as its referring to 2 requirements  i.e. CPU and memory

Reply:
Then we should remove `a`